docs/api.raml

```diff
@@ -0,0 +1,111 @@
+#%RAML 1.0
+---
+title: Platform Of Trust Translator
+version: v1
+baseUri: http://{base_url}/{version}
+mediaType: application/json
+types:
+  fetch-request:
+    type: object
+    properties:
+      timestamp:
+        description: The timestamp when the request was sent. RFC3339 format.
+        type: datetime
+        example: "2018-11-22T10:00:00Z"
+      productCode:
+        description: The product code of the data product.
+        type: string
+        example: example-room-temperature
+      parameters:
+        description: Optional additional parameters that the translator needs.
+        type: object
+        example: |
+          {
+            "deviceId": "r_305_d_32"
+          }
+/health:
+  description: API Health endpoints
+  get:
+    description: Health check endpoint
+    responses:
+      200:
+        body:
+          example: {}
+/fetch:
+  description: Fetch data from data source
+  post:
```

Comment:
Do you think it's possible to document incoming headers here?

Reply:
Yes, good that you noticed.